patterns/testimonials-review-large-image-right.php

```diff
@@ -0,0 +1,43 @@
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"},"spacing":{"padding":{"top":"0","bottom":"0","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|60"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"wrap","verticalAlignment":"space-between"}} -->
+<div class="wp-block-group" style="min-height:100%;padding-top:0;padding-right:var(--wp--preset--spacing--50);padding-bottom:0;padding-left:var(--wp--preset--spacing--50)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:heading {"className":"is-style-capsule","style":{"layout":{"selfStretch":"fit","flexSize":null}},"fontSize":"x-small"} -->
+<h2 class="wp-block-heading is-style-capsule has-x-small-font-size"><?php echo esc_html_x( 'What people are saying', 'Testimonial heading', 'twentytwentyfive' ); ?></h2>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:quote {"className":"is-style-plain","style":{"spacing":{"blockGap":"var:preset|spacing|50"}}} -->
+<blockquote class="wp-block-quote is-style-plain"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"left","contentSize":"400px"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.1"}},"fontSize":"x-large"} -->
+<p class="has-x-large-font-size" style="line-height:1.1"><?php echo esc_html_x( '&ldquo;Superb product and customer service!&rdquo;', 'Sample testimonial', 'twentytwentyfive' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --><cite><?php echo wp_kses_post( _x( 'Jo Mulligan <br /><sub>Atlanta, GA</sub>', 'Sample testimonial citation', 'twentytwentyfive' ) ); ?></cite></blockquote>
+<!-- /wp:quote --></div>
+<!-- /wp:group --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"}}}} -->
+<div class="wp-block-column" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-right:var(--wp--preset--spacing--50);padding-bottom:0;padding-left:var(--wp--preset--spacing--50)"><!-- wp:image {"aspectRatio":"1","scale":"cover","sizeSlug":"large","linkDestination":"none","align":"center"} -->
```

Comment:
Aspect ratio should be "Original", not Square.

Reply:
Here's what I'm seeing:

https://github.com/user-attachments/assets/a6eb8829-07fb-4e08-b38d-219fcbc0cbe8

Reply:
The image is set to "align center", it should be set to full to align with the wide width of the theme.

Reply:
> Aspect ratio should be "Original", not Square.

Do you think we should crop the source image in that case, or are you happy with this deviating from the [figma design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1810&node-type=SYMBOL&t=6iAKPXq8Uv9G9ZbM-0)? It would look quite different with "original" aspect ratio applied:
<img width="1135" alt="image" src="https://github.com/user-attachments/assets/72f4eca6-924e-4be4-aed3-d6f4196b539b">

> Here's what I'm seeing:
> 
>  image.mp4

This doesn't look right, I'm not seeing it locally (clean WP 6.6 with GB plugin enabled or disabled)
In your video it's visible that the image is overflowing the square bounding box, could something be interfering with it?

Reply:
Apologies, ignore my previous comment about the aspect ratio. Let's make sure it's **square**, but it's not looking zoomed in like my recording above and it looks like a bug.